scene/resources/primitive_meshes.cpp

```diff
@@ -1568,16 +1568,16 @@ void PrismMesh::_create_mesh_array(Array &p_arr) const {
 	thisrow = point;
 	prevrow = 0;
 	for (j = 0; j <= (subdivide_h + 1); j++) {
-		float v = j;
-		float v2 = j / (subdivide_h + 1.0);
-		v /= (2.0 * (subdivide_h + 1.0));
-
 		float left, right;
-		float scale = (y - start_pos.y) / size.y;
+		float scale = j / (subdivide_h + 1.0);
 
 		left = start_pos.x + (size.x * (1.0 - scale) * left_to_right);
 		right = left + (size.x * scale);
 
+		float v = j;
+		float v2 = scale;
+		v /= (2.0 * (subdivide_h + 1.0));
```

Comment:
Revove redundant parentheses:

```suggestion
		v /= 2.0 * (subdivide_h + 1.0);
```